bundles/InstallBundle/src/Installer.php

```diff
@@ -798,7 +798,7 @@ protected function insertDatabaseContents(Connection $db): void
     {
         $db->insert('assets', Helper::quoteDataIdentifiers($db, [
             'id' => 1,
-            'parentId' => 0,
+            'parentId' => null,
```

Comment:
@blankse What was the idea behind this change? Why is this necessary?

I would see that as a BC. If someone has a condition on an Listing to get the `parentId = 0` this would not work anymore.

Reply:
@martineiber When writing the test, I noticed that the ParentId is 0 after copying. I think it should be consistent with the other properties NULL. But yes changing the ParentIds on the root elements could be a BC break. I'll change that back.

Reply:
@martineiber Done :)